backend/cypress/support/check-access.js

```diff
@@ -1,23 +1,23 @@
 // reusable code for the accessandsubmission page
+const accessFields = [
+  '#certifying_auditee_contact_fullname', '#certifying_auditee_contact_email', '#certifying_auditee_contact_re_email',
+  '#certifying_auditor_contact_fullname', '#certifying_auditor_contact_email', '#certifying_auditor_contact_re_email',
+  '#auditee_contacts_fullname', '#auditee_contacts_email', '#auditee_contacts_re_email',
+  '#auditor_contacts_fullname', '#auditor_contacts_email', '#auditor_contacts_re_email',
+];
 
-export function addValidEmail(field) {
+export function addValidInfo(field) {
+  const fieldType = field.split('_').pop();
   cy.get(field)
     .clear()
-    .type('test.address-with+features@test.gsa.gov')
+    .type(fieldType === 'email' ? 'test.address-with+features@test.gsa.gov' : 'Percy A. Person')
     .blur();
 }
 
-function completeFormWithValidInfo() {
-  [
-    '#certifying_auditee_contact_email', '#certifying_auditee_contact_re_email',
-    '#certifying_auditor_contact_email', '#certifying_auditor_contact_re_email',
-    '#auditee_contacts_email', '#auditee_contacts_re_email',
-    '#auditor_contacts_email', '#auditor_contacts_re_email' 
-  ].forEach(field => addValidEmail(field))
-}
-
 export function testValidAccess() {
-  completeFormWithValidInfo();
+  cy.wrap(accessFields).each((field) => {
```

Comment:
`cy.wrap()` might not be necessary (or used correctly here), but I was having some trouble with the cypress commands executing in the right order. My understanding is that Cypress commands have their own special queueing system that kinda looks and feels like promises but isn't exactly the same and can cause issues with sequencing. This helped tests pass consistently, but I'm not committed to the idea!